From c64be9bdbd05085bd92824bf1320c1ce394e352d Mon Sep 17 00:00:00 2001
From: Uri Mashiach <uri.mashiach@compulab.co.il>
Date: Wed, 12 May 2021 11:17:31 +0300
Subject: [PATCH 110/113] iot-gate-imx8: enable USB to Ethernet driver SMSC
 LAN95xx

The relevant  interfaces are used in the baseboard and in the POEv2
extension module.

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 configs/iot-gate-imx8_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/iot-gate-imx8_defconfig b/configs/iot-gate-imx8_defconfig
index aa88e7956f..4d24268916 100644
--- a/configs/iot-gate-imx8_defconfig
+++ b/configs/iot-gate-imx8_defconfig
@@ -94,6 +94,7 @@ CONFIG_SDP_LOADADDR=0x40400000
 CONFIG_USB_GADGET_DOWNLOAD=y
 CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX88179=y
+CONFIG_USB_ETHER_SMSC95XX=y
 CONFIG_EXT4_WRITE=y
 CONFIG_FAT_WRITE=y
 CONFIG_OF_LIBFDT_OVERLAY=y
-- 
2.11.0

